### PR TITLE
Add sabnzbd plugin

### PR DESCRIPTION
**Pull request by RogerAirgood**
This adds the Sabnzbd plugin. Sabnzbd is a popular newsgroup reader.

I've tested this via iocage on the CLI and in the UI on 11.3-RC2. After installation the user is greeted with a Web UI to continue the install.

### CODE REVIEW

File: sabnzbd.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+    "name": "sabnzbd",
+    "release": "11.3-RELEASE",
+    "artifact": "https://github.com/RogerAirgood/iocage-plugin-sabnzbd.git",
+    "properties": {
+        "nat": 1,
+        "nat_forwards": "tcp(8080:8080)"
+    },
+    "pkgs": [
+	      "sabnzbdplus"
```
Comment (sonicaj) [inline sabnzbd.json]: Can we please correct the indentation for this line ?

Comment (RogerAirgood) [reply sabnzbd.json]: I've corrected the indent, thanks.

**Comment by miwi-fbsd** [comment]
Hi,

Thanks for you submission, can you please have a look at https://github.com/RogerAirgood/iocage-plugin-sabnzbd/pull/1 , I have cleanup the script a bit which makes it a bit more robust.

Once that is in your repo, we are good to merge it in :-).

**Comment by RogerAirgood** [comment]
I've merged your changes @miwi-fbsd Thank you for the help.

**Comment by miwi-fbsd** [comment]
Thanks.

**Pull request merged**